core/unit_test/TestGraph.hpp

```diff
@@ -113,6 +113,10 @@ TEST_F(TEST_CATEGORY_FIXTURE(graph), launch_one_rvalue) {
 }
 
 TEST_F(TEST_CATEGORY_FIXTURE(graph), launch_six) {
+#ifdef KOKKOS_ENABLE_OPENMPTARGET  // FIXME_OPENMPTARGET team_size incompatible
+  if (std::is_same_v<TEST_EXECSPACE, Kokkos::Experimental::OpenMPTarget>)
+    GTEST_SKIP() << "skipping since OpenMPTarget can't use team_size 1";
```

Comment:
And that team size cannot be easily set to an acceptable value?  Why did you revert your initial attention to set it to 32?  What was the issue?

Reply:
The test was failing with a larger team size (and I didn't investigate more after that).

Reply:
Fine but the comment does not really capture that you tried the usual "if using openmptarget then set a team size of 32" and that it did not resolve the issue.